src/task/syncNotifications.ts

```diff
@@ -102,7 +102,8 @@ async function executor(db: Tnex, job: JobLogger) {
     }
     ++progress;
     job.setProgress(progress / len, undefined);
-  }
+  });
+  await Promise.all(promises);
```

Comment:
Does our library handle rate limiting to ESI for us? Is that something we need to worry about? IIRC ESI doesn't care much about rate limiting for read endpoints (unlike, say, the zkill api), but I haven't checked recently.

Reply:
https://developers.eveonline.com/blog/article/error-limiting-imminent -- 60 second rolling window, but we execute every 60 sec regardless so